spacy/tests/pipeline/test_entity_ruler.py

```diff
@@ -90,6 +91,17 @@ def test_entity_ruler_clear(nlp, patterns):
     assert len(doc.ents) == 0
 
 
+def test_entity_ruler_fix8216(nlp, patterns):
```

Comment:
Test looks good to me, but as it's a regression test tied to a specific issue, I'd put it in the `tests/regression` folder :-)

Reply:
Ah, thanks for the tip - I didn't realize that existed.

Reply:
And done.